tests/utils/actions/keyboard.ts

```diff
@@ -28,7 +28,7 @@ export async function type(page: Page, content: string, delay = 50) {
 export async function withPressKey(
   page: Page,
   key: string,
-  fn: () => Promise<void>
+  fn: () => Promise<void> = () => new Promise(resolve => resolve())
```

Comment:
This method is designed to run the handler, so if you don't want to pass in the callback, using `pressBackspace` will be semantically a better fit.

Reply:
yeah, I should have found there is pressBackspace method.